tooling/api/src/path.ts

```diff
@@ -529,7 +582,16 @@ async function videoDir(): Promise<string> {
 }
 
 /**
- * Returns the path to the suggested log directory.
+ * Alias of {@link appLogDir} for backwards-compatibility purposes.
+ *
+ * @since 1.0.0
+ */
```

Comment:
keep the original doc comment here and add `@deprecation` notice saying this will be removed in v2 and suggest using the new function